src/main/io/osd.c

```diff
@@ -1015,8 +1015,25 @@ static void osdDisplayStatisticLabel(uint8_t y, const char * text, const char *
     displayWrite(osdDisplayPort, 22, y, value);
 }
 
+/*
```

Comment:
The comment isn't strictly needed, the name of the function is pretty much self explanatory. ;-)